src/javascripts/ng-admin/Crud/delete/DeleteController.js

```diff
@@ -17,6 +17,7 @@ define(function () {
         this.notification = notification;
         this.$scope.entry = entry;
         this.$scope.view = view;
+        this.goBack = function(){ $window.history.back(); }
```

Comment:
please make it a property of the controller prototype instead, like other functions.